### ISSUE TITLE
input btn 위치

### ISSUE THREAD

**Issue by Sooamazing** [abtinia/Groom_PBL#3]
https://github.com/abtinia/Groom_PBL/blob/cd88d17251f8235f886f766431f8cc611c80fcc9/TODO_App/To-Do-App/script.js#L1

CSS이긴 하지만.... 여기에 한 번에 하는 게 좋을 거 같아서 여기에 씁니당
저 btn이 input 안으로 어떻게 오는지 궁금했는데, container 안에 input, btn을 각각 준다는 걸 덕분에 알아갑니다!!!!!

**Comment by abtinia** [comment]
네, btn이 add 버튼을 의미하신 거라면 그냥 container 내부에 input, btn 각각 주고 대신 그 둘을 감싸주는 div태그를 하나 만들어서 
그 div태그를 flex로 위치 조정했습니다.
그리고 리뷰 이렇게 한 파일에 몰아서 해주신 부분.. 일부러 배려해주시는 부분.. 디테일 너무 아름답구요
감사드립니다 ㅋㅋ 저도 되도록이면 한나님이 하신것처럼 해야겠어요..
